src/Plugin/Field/FieldFormatter/StrawberryPagedFormatter.php

```diff
@@ -407,6 +422,7 @@ public function processElementforMetadatadisplays($delta, array $jsondata, Field
     $max_width = $this->getSetting('max_width');
     $max_width_css = empty($max_width) || $max_width == 0 ? '100%' : $max_width .'px';
     $max_height = $this->getSetting('max_height');
+    $ia_reader_images_base_url = $this->getSetting('ia_reader_images_base_url');
```

Comment:
The logic on 'iareaderimagesbaseurl' => strlen($ia_reader_images_base_url) > 0 ? $ia_reader_images_base_url : $this->getIABookReaderImagesDefaultBaseURL(),
Should go here. And you can also check/add/strip/restore the last "/" to be sure it always ends in a single "/" in the same line.